### PR TITLE
Recipes: Cluster Proxy/Agent

### PR DESCRIPTION
**Pull request by anipenguin**
Drafted and Uploaded 6 Recipes for ClusterProxy/Agent in response to PR:
- https://github.com/sporeprotocol/spore-sdk/pull/45

In this PR:
- [x] Create Cluster Proxy
- [x] Create Cluster Agent
- [x] Transfer Cluster Proxy
- [x] Transfer Cluster Agent
- [x] Melt Cluster Proxy/Agent
- [x] Reordered Recipes in menu
- [x] Updated some links to code examples

### CODE REVIEW

**Comment by ashuralyk** [comment]
@ShookLyngs please take your time to help me check this, thx

File: docs/recipes/create-cluster-agent.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Cluster Proxy and Cluster Agents offer a solution to verify ownership when referencing a [cluster](/glossary/terms#cluster) during simultaneous spore creation.
+
+You will learn how to create a [Cluster Agent](/glossary/terms#cluster-agent), which serves as a representative for a Cluster. This enables the creation of [clustered spores](/recipes/create-clustered-spore) without requiring the signatures or approvals from the owner of the Cluster being represented.
+
+## Create a Cluster Agent
+
+### Method 1
+
+You can create a Cluster Agent by referencing a [ClusterProxy](/recipes/create-cluster-proxy) with the [`createClusterAgent` API](https://github.com/sporeprotocol/spore-sdk/blob/beta/docs/core/composed-apis.md#createclusteragent) from spore-sdk:
```
Comment (ashuralyk) [inline docs/recipes/create-cluster-agent.mdx:17]: @ShookLyngs this doc from `spore-sdk` repo is outdated or not? please confirm.

Comment (ShookLyngs) [reply docs/recipes/create-cluster-agent.mdx:17]: The composed-apis doc should be up-to-date, and it should be updated if it's not.

Comment (ashuralyk) [reply docs/recipes/create-cluster-agent.mdx:17]: > The composed-apis doc should be up-to-date, and it should be updated if it's not.

the 'should' means 'already'?

**Comment by ashuralyk** [comment]
I approved that not to block work

**Pull request merged**